### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by ziyadbasheer**
``` release-note
* Rewrote two sentences.
```

### CODE REVIEW

**Comment by ziyadbasheer** [comment]
I signed it!

**Comment by ziyadbasheer** [comment]
@ixdy @k8s-bot is spammy. Is it because this is my very first contribution?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ Want to hack on kubernetes? Yay!
 
 ## Developer Guide
 
-We have a [Developer's Guide](docs/devel/README.md) that outlines everything you need to know from setting up your dev environment to how to get faster Pull Request reviews. And of course if you find something undocumented or incorrect along the way, feel free to send a PR!
+We have a [Developer's Guide](docs/devel/README.md) that outlines everything you need to know from setting up your dev environment to how to get faster Pull Request reviews. If you find something undocumented or incorrect along the way, please feel free to send a Pull Request.
```
Comment (eparis) [inline CONTRIBUTING.md]: We use the phrase `PR` many places. Could/should we say `... to send a Pull Request (PR).` So they can some context for the acronym in later reading?

Comment (ziyadbasheer) [inline CONTRIBUTING.md]: I did notice the use of "PR" in several of the other documents that I read. My argument is that being new to Kubernetes development, I would much rather reserve every bit of mental energy to understanding the code-base, not deciphering unnecessary abbreviations. 

I'll do the grunt-work of reviewing the documentation and replacing every single instance of PR with Pull Request, making minor copy edits as I go along and send another _Pull Request_.

**Comment by eparis** [comment]
ok to test

**Pull request merged**